setup.py

```diff
@@ -569,15 +573,31 @@ def srcpath(name=None, suffix=".pyx", subdir="src"):
     include = data.get("include", [])
     include.append(numpy.get_include())
 
+    extra_comp_args = extra_compile_args.copy()
+    comp_macros = data.get("macros", macros)
+    undef_macros = []
+
+    if is_platform_zos():
+        language = data.get("language", None)
+        if language == "c++":
+            compiler = os.environ.get("CXX", "/bin/xlc++")
+            compiler_name = os.path.basename(compiler)
+
+            if (compiler_name == "xlc") or (compiler_name == "xlc++"):
```

Comment:
Feels that all these if could be just a small condition:
```python
if sys.platform == "zos" and data.get('language') == 'c++' and os.path.basename(os.environ.get('CXX')) in ('xlc', 'xlc++'):
```
Does it make sense?

Reply:
Yup - I've made this change. Thanks